src/app/warningmessage/warningmessage.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+
+ 
+  
+@Component({
+  selector: 'warningmessage',
+  templateUrl: './warningmessage.component.html',
```

Comment:
you cannot have both templateUrl and template defined for the meta-data here. you need to mention only one.
As per instructions, use inline template